RGenGC のデバッグ用のチェックで rb_bug() で異常終了させる時のエラーメッセージの
修正。
